src/tests/ftest/deployment/io_sys_admin.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/python
+"""
+  (C) Copyright 2018-2021 Intel Corporation.
+
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+"""
+
+import time
+
+from file_count_test_base import FileCountTestBase
+from data_mover_test_base import DataMoverTestBase
+from general_utils import human_to_bytes
+from pool_test_base import PoolTestBase
+import security_test_base as secTestBase
+
+class IoSysAdmin(DataMoverTestBase, FileCountTestBase):
+    # pylint: disable=too-many-ancestors
+    """Test Class Description: Test class for large IO tests and some
+                               basic system admin operations.
+    :avocado: recursive
+    """
+
+    def get_free_space(self):
+        """Display pool free space."""
+        free_space = []
+        scm_free_space = self.pool.get_pool_free_space("scm")
+        free_space.append(scm_free_space)
+        nvme_free_space = self.pool.get_pool_free_space("nvme")
+        free_space.append(nvme_free_space)
+        self.log.info("Free space [SCM, Nvme]: [%s, %s]", scm_free_space, nvme_free_space)
+
+        return free_space
+
+    def test_io_sys_admin(self):
+        """
+        Test Description: Performs tests to generate large data sets over
+                          various middleware, perform various system admin
+                          operations, datamover operations.
+        :avocado: tags=hw,large
+        :avocado: tags=deployment,iosysadmin
+        """
+        # local param
+#        ec_ior_flags = self.params.get("ec_ior_flags", "/run/ior/*")
+#        hdf5_plugin_path = self.params.get("plugin_path", '/run/hdf5_vol/*')
+        new_test_user = self.params.get("new_user", "/run/container_acl/*")
+        new_test_group = self.params.get("new_group", "/run/container_acl/*")
+
+        dmg = self.get_dmg_command()
+        daos = self.get_daos_command()
+
+        secTestBase.add_del_user(
+            self.hostlist_clients, "useradd", new_test_user)
+        secTestBase.add_del_user(
+            self.hostlist_clients, "groupadd", new_test_group)
+
+        for idx in range(1, 4):
+            self.add_pool_qty(1, namespace="/run/pool_{}/".format(idx), create=False)
+            PoolTestBase.check_pool_creation(self, 60)
+            self.pool[-1].connect()
+            self.add_container_qty(1, self.pool[-1],
+                                   namespace="/run/container_{}/".format(idx), create=False)
+            for _, container in enumerate(self.container):
+                container.create()
+                daos.container_set_owner(self.pool[-1].uuid, self.container[-1].uuid,
+                                         new_test_user, new_test_group)
```

Comment:
For 3 containers per pool:
```suggestion
            for cont_idx in range(1, 4):
                self.add_container_qty(1, self.pool[-1], namespace="/run/container_{}/".format(cont_idx))
                daos.container_set_owner(self.pool[-1].uuid, self.container[-1].uuid,
                                         new_test_user, new_test_group)
```

Reply:
Done